--- extras/prj/build.lst.orig	2005-05-20 18:46:00.000000000 +0200
+++ extras/prj/build.lst	2005-12-30 10:37:17.750718900 +0100
@@ -13,6 +13,34 @@
 er      extras\source\gallery\sounds                            nmake   -       all     er_gsnds NULL
 er      extras\source\gallery\www-back                          nmake   -       all     er_gwbck NULL
 er      extras\source\gallery\www-graf                          nmake   -       all     er_gwgrf NULL
+er      extras\source\gallery\accessories                       nmake   -       all     er_gacce NULL
+er      extras\source\gallery\accessories\Draws\People          nmake   -       all     er_gadpe NULL
+er      extras\source\gallery\accessories\Photos\Buildings      nmake   -       all     er_gapbu NULL
+er      extras\source\gallery\accessories\Photos\Celebration    nmake   -       all     er_gapce NULL
+er      extras\source\gallery\accessories\Photos\Cities         nmake   -       all     er_gapci NULL
+er      extras\source\gallery\accessories\Photos\Fauna          nmake   -       all     er_gapfa NULL
+er      extras\source\gallery\accessories\Photos\Flowers        nmake   -       all     er_gapfl NULL
+er      extras\source\gallery\accessories\Photos\FoodsandDrinks nmake   -       all     er_gapfo NULL
+er      extras\source\gallery\accessories\Photos\Humans         nmake   -       all     er_gaphu NULL
+er      extras\source\gallery\accessories\Photos\Landscapes     nmake   -       all     er_gapla NULL
+er      extras\source\gallery\accessories\Photos\Objects        nmake   -       all     er_gapob NULL
+er      extras\source\gallery\accessories\Photos\Plants         nmake   -       all     er_gappl NULL
+er      extras\source\gallery\accessories\Photos\Space          nmake   -       all     er_gapsp NULL
+er      extras\source\gallery\accessories\Photos\Statues        nmake   -       all     er_gapst NULL
+er      extras\source\gallery\accessories\Photos\Travel         nmake   -       all     er_gaptr NULL
+er      extras\source\premium\templates\layout\lang             nmake   -       all     er_prtela NULL
+er      extras\source\premium\templates\presnt\lang             nmake   -       all     er_prtepr NULL
+er      extras\source\premium\templates\educate\lang            nmake   -       all     er_prteed NULL
+er      extras\source\premium\templates\finance\lang            nmake   -       all     er_prtefi NULL
+er      extras\source\premium\templates\forms\lang              nmake   -       all     er_prtefo NULL
+er      extras\source\premium\templates\misc\lang               nmake   -       all     er_prtemi NULL
+er      extras\source\premium\templates\officorr\lang           nmake   -       all     er_prteoc NULL
+er      extras\source\premium\templates\offimisc\lang           nmake   -       all     er_prteom NULL
+er      extras\source\premium\templates\personal\lang           nmake   -       all     er_prtepe NULL
+er      extras\source\premium\templates\lang                    nmake   -       all     er_prtemp NULL
+er      extras\source\premium\samples\advertisement\lang        nmake   -       all     er_prsaad NULL
+er      extras\source\premium\samples\documentation\lang        nmake   -       all     er_prsedo NULL
+er      extras\source\premium\samples\lang                      nmake   -       all     er_prsamp NULL
 er	extras\source\misc_config				nmake	-	all	er_mcnf NULL
 er      extras\source\misc_config\lang                          nmake   -       all     er_mcnfl er_mcnf NULL
 er	extras\source\palettes\lang				nmake	-	all	er_pall er_mcnfl NULL
