--- svx/source/gallery2/galtheme.src.orig	2006-10-16 16:39:54.093750000 +0200
+++ svx/source/gallery2/galtheme.src	2006-10-16 16:39:20.390625000 +0200
@@ -358,42 +358,1025 @@
 	Text = "Dummy 5";
 };
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_SHAPES_POLYGONS
+{
+Text [ en-US ] = "Shapes - polygons";
+Text [ de ] = "Formen Polygone";
+Text [ hu ] = "Alakzatok - poligon";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_SHAPES_1
+{
+Text [ en-US ] = "Shapes 1";
+Text [ de ] = "Formen 1";
+Text [ hu ] = "Alakzatok 1";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_SHAPES_2
+{
+Text [ en-US ] = "Shapes 2";
+Text [ de ] = "Formen 2";
+Text [ hu ] = "Alakzatok 2";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ANIMALS
+{
+Text [ en-US ] = "Animals";
+Text [ de ] = "Tiere";
+Text [ hu ] = "Állatok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CARS
+{
+Text [ en-US ] = "Cars";
+Text [ de ] = "Autos";
+Text [ hu ] = "Autók";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_BUGS
+{
+Text [ en-US ] = "Bugs";
+Text [ de ] = "Insekten";
+Text [ hu ] = "Bogarak";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CISCO_OTHER
+{
+Text [ en-US ] = "Cisco - Other";
+Text [ de ] = "Cisco - Andere";
+Text [ hu ] = "Cisco - Egyéb";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CISCO_MEDIA
+{
+Text [ en-US ] = "Cisco - Media";
+Text [ de ] = "Cisco Medien";
+Text [ hu ] = "Cisco - Média";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CISCO_PRODUCTS
+{
+Text [ en-US ] = "Cisco - Products";
+Text [ de ] = "Cisco Produkte";
+Text [ hu ] = "Cisco - Termékek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CISCO_WAN_LAN
+{
+Text [ en-US ] = "Cisco - WAN - LAN";
+Text [ de ] = "Cisco - WAN - LAN";
+Text [ hu ] = "Cisco - WAN - LAN";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_DOMINO_USUAL
+{
+Text [ en-US ] = "Domino - usual";
+Text [ de ] = "Domino - normal";
+Text [ hu ] = "Dominó - hagyományos";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_DOMINO_NUMBERED
+{
+Text [ en-US ] = "Domino - numbered";
+Text [ de ] = "Domino - nummeriert";
+Text [ hu ] = "Dominó - számozott";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ELECTRONICS_PARTS_1
+{
+Text [ en-US ] = "Electronics - parts 1";
+Text [ de ] = "Elektronik - Teil 1";
+Text [ hu ] = "Elektronika - alkatrészek 1";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ELECTRONICS_PARTS_2
+{
+Text [ en-US ] = "Electronics - parts 2";
+Text [ de ] = "Elektronik - Teil 2";
+Text [ hu ] = "Elektronika - alkatrészek 2";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ELECTRONICS_PARTS_3
+{
+Text [ en-US ] = "Electronics - parts 3";
+Text [ de ] = "Elektronik - Teil 3";
+Text [ hu ] = "Elektronika - alkatrészek 3";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ELECTRONICS_PARTS_4
+{
+Text [ en-US ] = "Electronics - parts 4";
+Text [ de ] = "Elektronik - Teil 4";
+Text [ hu ] = "Elektronika - alkatrészek 4";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ELECTRONICS_CIRCUIT
+{
+Text [ en-US ] = "Electronics - circuit";
+Text [ de ] = "Elektronik - Schaltkreise";
+Text [ hu ] = "Elektronika - áramkörök";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ELECTRONICS_SIGNS
+{
+Text [ en-US ] = "Electronics - signs";
+Text [ de ] = "Elektronik - Symbole";
+Text [ hu ] = "Elektronika - jelek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ELECTRONICS_GAUGES
+{
+Text [ en-US ] = "Electronics - gauges";
+Text [ de ] = "Elektronik - Pegel";
+Text [ hu ] = "Elektronika - mérõeszközök";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PEOPLE_1
+{
+Text [ en-US ] = "People";
+Text [ de ] = "Menschen";
+Text [ hu ] = "Emberek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ARCHITECTURE_OVERLAY
+{
+Text [ en-US ] = "Architecture - overlay";
+Text [ de ] = "Architektur - Überleger";
+Text [ hu ] = "Építészet - burkolat";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ARCHITECTURE_FURNITURES
+{
+Text [ en-US ] = "Architecture - furnitures";
+Text [ de ] = "Architektur - Möbel";
+Text [ hu ] = "Építészet - butorok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ARCHITECTURE_BUILDINGS
+{
+Text [ en-US ] = "Architecture - buildings";
+Text [ de ] = "Architektur - Gebäude";
+Text [ hu ] = "Építészet - épületek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ARCHITECTURE_BATHROOM_KITCHEN
+{
+Text [ en-US ] = "Architecture - bathroom, kitchen";
+Text [ de ] = "Architektur - Badezimmer, Küche";
+Text [ hu ] = "Építészet - fürdõszoba, konyha";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ARCHITECTURE_KITCHEN
+{
+Text [ en-US ] = "Architecture - kitchen";
+Text [ de ] = "Architektur - Küche";
+Text [ hu ] = "Építészet - konyha";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ARCHITECTURE_WINDOWS_DOORS
+{
+Text [ en-US ] = "Architecture - windows, doors";
+Text [ de ] = "Architektur - Fenster, Türen";
+Text [ hu ] = "Építészet - nyílászáró";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_FLOWCHARTS_1
+{
+Text [ en-US ] = "Flowcharts";
+Text [ de ] = "Flußdiagramme";
+Text [ hu ] = "Folyamatábra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_FLOWCHARTS_2
+{
+Text [ en-US ] = "Flowcharts 2";
+Text [ de ] = "Flußdiagramme 2";
+Text [ hu ] = "Folyamatábra 2";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_FORALSTUDIOA
+{
+Text [ en-US ] = "Foral-StudioA";
+Text [ de ] = "Foral-StudioA";
+Text [ hu ] = "Foral-StudioA";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_FAUNA
+{
+Text [ en-US ] = "Photos - Fauna";
+Text [ de ] = "Fotos - Tiere";
+Text [ hu ] = "Fotók - Élõvilág";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_BUILDINGS
+{
+Text [ en-US ] = "Photos - Buildings";
+Text [ de ] = "Fotos - Gebäude";
+Text [ hu ] = "Fotók - Épületek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_PLANTS
+{
+Text [ en-US ] = "Photos - Plants";
+Text [ de ] = "Fotos - Pflanzen";
+Text [ hu ] = "Fotók - Növények";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_STATUES
+{
+Text [ en-US ] = "Photos - Statues";
+Text [ de ] = "Fotos - Statuen";
+Text [ hu ] = "Fotók - Szobrok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_LANDSCAPES
+{
+Text [ en-US ] = "Photos - Landscapes";
+Text [ de ] = "Fotos - Landschaften";
+Text [ hu ] = "Fotók - Tájak";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_CITIES
+{
+Text [ en-US ] = "Photos - Cities";
+Text [ de ] = "Fotos - Städte";
+Text [ hu ] = "Fotók - Városok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_FLOWERS
+{
+Text [ en-US ] = "Photos - Flowers";
+Text [ de ] = "Fotos - Blumen";
+Text [ hu ] = "Fotók - Virágok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_THERAPEUTICS_GENERAL
+{
+Text [ en-US ] = "Therapeutics - general";
+Text [ de ] = "Medizin - Allgemein";
+Text [ hu ] = "Gyógyászat - általános";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_WEATHER
+{
+Text [ en-US ] = "Weather";
+Text [ de ] = "Wetter";
+Text [ hu ] = "Idõjárás";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_VECHILES
+{
+Text [ en-US ] = "Vehicles";
+Text [ de ] = "Fahrzeuge";
+Text [ hu ] = "Jármûvek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_SIGNS
+{
+Text [ en-US ] = "Signs";
+Text [ de ] = "Zeichen";
+Text [ hu ] = "Jelek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_BLUE_MAN
+{
+Text [ en-US ] = "Blue Man";
+Text [ de ] = "Blaue Figuren";
+Text [ hu ] = "Kék ember";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CHEMISTRY_AMINO_ACIDS
+{
+Text [ en-US ] = "Chemistry - Amino acids";
+Text [ de ] = "Chemie - Aminosäuren";
+Text [ hu ] = "Kémia - Aminósavak";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_LOGICAL_SIGNS
+{
+Text [ en-US ] = "Logical signs";
+Text [ de ] = "Logische Symbole";
+Text [ hu ] = "Logikai jelek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_LOGICAL_GATES
+{
+Text [ en-US ] = "Logical gates";
+Text [ de ] = "Logische Gatter";
+Text [ hu ] = "Logikai kapuk";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_LOGOS
+{
+Text [ en-US ] = "Logos";
+Text [ de ] = "Logos";
+Text [ hu ] = "Logók";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_SMILES
+{
+Text [ en-US ] = "Smilies";
+Text [ de ] = "Smilies";
+Text [ hu ] = "Mosolyok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_ARROWS_1
+{
+Text [ en-US ] = "Arrows";
+Text [ de ] = "Pfeile";
+Text [ hu ] = "Nyilak";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_01_CLOCK
+{
+Text [ en-US ] = "Clock - 01 clock";
+Text [ de ] = "Uhr - 01 Uhr";
+Text [ hu ] = "Óra - 01 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_02_CLOCK
+{
+Text [ en-US ] = "Clock - 02 clock";
+Text [ de ] = "Uhr - 02 Uhr";
+Text [ hu ] = "Óra - 02 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_03_CLOCK
+{
+Text [ en-US ] = "Clock - 03 clock";
+Text [ de ] = "Uhr - 03 Uhr";
+Text [ hu ] = "Óra - 03 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_04_CLOCK
+{
+Text [ en-US ] = "Clock - 04 clock";
+Text [ de ] = "Uhr - 04 Uhr";
+Text [ hu ] = "Óra - 04 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_05_CLOCK
+{
+Text [ en-US ] = "Clock - 05 clock";
+Text [ de ] = "Uhr - 05 Uhr";
+Text [ hu ] = "Óra - 05 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_06_CLOCK
+{
+Text [ en-US ] = "Clock - 06 clock";
+Text [ de ] = "Uhr - 06 Uhr";
+Text [ hu ] = "Óra - 06 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_07_CLOCK
+{
+Text [ en-US ] = "Clock - 07 clock";
+Text [ de ] = "Uhr - 07 Uhr";
+Text [ hu ] = "Óra - 07 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_08_CLOCK
+{
+Text [ en-US ] = "Clock - 08 clock";
+Text [ de ] = "Uhr - 08 Uhr";
+Text [ hu ] = "Óra - 08 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_09_CLOCK
+{
+Text [ en-US ] = "Clock - 09 clock";
+Text [ de ] = "Uhr - 09 Uhr";
+Text [ hu ] = "Óra - 09 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_10_CLOCK
+{
+Text [ en-US ] = "Clock - 10 clock";
+Text [ de ] = "Uhr - 10 Uhr";
+Text [ hu ] = "Óra - 10 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_11_CLOCK
+{
+Text [ en-US ] = "Clock - 11 clock";
+Text [ de ] = "Uhr - 11 Uhr";
+Text [ hu ] = "Óra - 11 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_CLOCK_12_CLOCK
+{
+Text [ en-US ] = "Clock - 12 clock";
+Text [ de ] = "Uhr - 12 Uhr";
+Text [ hu ] = "Óra - 12 óra";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PNEUMATIC_PARTS
+{
+Text [ en-US ] = "Pneumatic - parts";
+Text [ de ] = "Pneumatik - Teile";
+Text [ hu ] = "Pneumatika - alkatrészek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_COMPUTER_GENERAL
+{
+Text [ en-US ] = "Computer - general";
+Text [ de ] = "Computer - Allgemein";
+Text [ hu ] = "Számítástechnika - általános";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_COMPUTER_NETWORK
+{
+Text [ en-US ] = "Computer - network";
+Text [ de ] = "Computer - Netzwerk";
+Text [ hu ] = "Számítástechnika - hálózat";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_COMPUTER_NETWORK_DEVICES
+{
+Text [ en-US ] = "Computer - network devices";
+Text [ de ] = "Computer - Netzwerk Geräte";
+Text [ hu ] = "Számítástechnika - Hálózati elemek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_COMPUTER_WIFI
+{
+Text [ en-US ] = "Computer - WIFI";
+Text [ de ] = "Computer - WIFI";
+Text [ hu ] = "Számítástechnika - WIFI";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_NUMBERS
+{
+Text [ en-US ] = "Numbers";
+Text [ de ] = "Nummern";
+Text [ hu ] = "Számok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_SIGNS_DANGER
+{
+Text [ en-US ] = "Signs - danger";
+Text [ de ] = "Gefahrenzeichen";
+Text [ hu ] = "Táblák - Veszély";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_OBJECTS
+{
+Text [ en-US ] = "Objects";
+Text [ de ] = "Objekte";
+Text [ hu ] = "Tárgyak";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_MAPS_AFRICA
+{
+Text [ en-US ] = "Maps - Africa";
+Text [ de ] = "Karten - Afrika";
+Text [ hu ] = "Térkép - Afrika";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_UNITED_STATES_OF_AMERICA
+{
+Text [ en-US ] = "Maps - United States of America";
+Text [ de ] = "Karten - Vereinigte Staaten von Amerika";
+Text [ hu ] = "Térkép - Amerikai Egyesült Államok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_MAPS_AUSTRALIA
+{
+Text [ en-US ] = "Maps - Australia";
+Text [ de ] = "Karten - Australien";
+Text [ hu ] = "Térkép - Ausztrália";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_MAPS_ASIA
+{
+Text [ en-US ] = "Maps - Asia";
+Text [ de ] = "Karten - Asien";
+Text [ hu ] = "Térkép - Ázsia";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_MAPS_SOUTH_AMERICA
+{
+Text [ en-US ] = "Maps - South America";
+Text [ de ] = "Karten - Südamerika";
+Text [ hu ] = "Térkép - Dél-Amerika";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_MAPS_EUROPE
+{
+Text [ en-US ] = "Maps - Europe";
+Text [ de ] = "Karten - Europa";
+Text [ hu ] = "Térkép - Európa";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_MAPS_EUROPE_1
+{
+Text [ en-US ] = "Maps - Europe 1";
+Text [ de ] = "Karten - Europa 1";
+Text [ hu ] = "Térkép - Európa 1";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_FRANCE
+{
+Text [ en-US ] = "Maps - France";
+Text [ de ] = "Karten - Frankreich";
+Text [ hu ] = "Térkép - Franciaország";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_FRANCE_COUNTRIES
+{
+Text [ en-US ] = "Maps - France - countries";
+Text [ de ] = "Karten - Französische Länder";
+Text [ hu ] = "Térkép - Franciaország - régiók";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_SIGNS
+{
+Text [ en-US ] = "Maps - signs";
+Text [ de ] = "Karten - Symbole";
+Text [ hu ] = "Térkép - jelek";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_CANADA
+{
+Text [ en-US ] = "Maps - Canada";
+Text [ de ] = "Karten - Kanada";
+Text [ hu ] = "Térkép - Kanada";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_CONTINENTS
+{
+Text [ en-US ] = "Maps - Continents";
+Text [ de ] = "Karten - Kontinente";
+Text [ hu ] = "Térkép - kontinensek";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_MIDDLE_EAST
+{
+Text [ en-US ] = "Maps - Middle East";
+Text [ de ] = "Karten - Mittlerer Osten";
+Text [ hu ] = "Térkép - Közel-Kelet";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_MIDDLE_AMERICA
+{
+Text [ en-US ] = "Maps - Middle America";
+Text [ de ] = "Karten - Mittelamerika";
+Text [ hu ] = "Térkép - Közép-Amerika";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_MIDDLE_AGES
+{
+Text [ en-US ] = "Maps - Middle ages";
+Text [ de ] = "Karten - Mittelalter";
+Text [ hu ] = "Térkép - Középkor";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_MEXICO
+{
+Text [ en-US ] = "Maps - Mexico";
+Text [ de ] = "Karten - Mexiko";
+Text [ hu ] = "Térkép - Mexikó";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_ANCIENT_TIMES
+{
+Text [ en-US ] = "Maps - Ancient times";
+Text [ de ] = "Karten - frühere Zeiten";
+Text [ hu ] = "Térkép - Ókor";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_SIMBOLS
+{
+Text [ en-US ] = "Maps - simbols";
+Text [ de ] = "Karten - Symbole";
+Text [ hu ] = "Térkép - rajzjelek";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_HISTORY_1900
+{
+Text [ en-US ] = "Maps - history - 1900";
+Text [ de ] = "Karten - Geschichte - 1900";
+Text [ hu ] = "Térkép - történelem - 1900";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MAPS_WORLD
+{
+Text [ en-US ] = "Maps - World";
+Text [ de ] = "Karten - Welt";
+Text [ hu ] = "Térkép - Világ";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_CROPS
+{
+Text [ en-US ] = "Crops";
+Text [ de ] = "Obst und Gemüse";
+Text [ hu ] = "Termények";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_FRACTIONS
+{
+Text [ en-US ] = "Fractions";
+Text [ de ] = "Diagramme Torten und Balken";
+Text [ hu ] = "Törtek";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_FLAGS_1
+{
+Text [ en-US ] = "Flags";
+Text [ de ] = "Flaggen";
+Text [ hu ] = "Zászlók";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------
 
+String RID_GALLERYSTR_THEME_MUSIC_INSTRUMENTS
+{
+Text [ en-US ] = "Music - instruments";
+Text [ de ] = "Musikinstrumente";
+Text [ hu ] = "Zene - hangszer";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_MUSIC_SHEET_MUSIC
+{
+Text [ en-US ] = "Music - sheet music";
+Text [ de ] = "Notensatzzeichen";
+Text [ hu ] = "Zene - kotta";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_SPECIAL_PICTOGRAMM
+{
+Text [ en-US ] = "Special Pictogramms";
+Text [ de ] = "Speziell Pictogramm";
+Text [ hu ] = "Speciális piktogramok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_CELEBRATION
+{
+Text [ en-US ] = "Photos - Celebration";
+Text [ de ] = "Fotos - Feier";
+Text [ hu ] = "Fotók - Ünnepek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_FOODSANDDRINKS
+{
+Text [ en-US ] = "Photos - Foods and Drinks";
+Text [ de ] = "Fotos - Essen und Trink";
+Text [ hu ] = "Fotók - Ételek és italok";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_HUMANS
+{
+Text [ en-US ] = "Photos - Humans";
+Text [ de ] = "Fotos - Leute";
+Text [ hu ] = "Fotók - Emberek";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_OBJECTS
+{
+Text [ en-US ] = "Photos - Objects";
+Text [ de ] = "Fotos - Objekt";
+Text [ hu ] = "Fotók - Tárgyak";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_SPACE
+{
+Text [ en-US ] = "Photos - Space";
+Text [ de ] = "Fotos - Weltraum";
+Text [ hu ] = "Fotók - Világűr";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_PHOTOS_TRAVEL
+{
+Text [ en-US ] = "Photos - Travel";
+Text [ de ] = "Fotos - Fahrt";
+Text [ hu ] = "Fotók - Utazás";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_OPENOFFICEORG_LOGOS
+{
+Text [ en-US ] = "OpenOffice.org logos";
+Text [ de ] = "OpenOffice.org logos";
+Text [ hu ] = "OpenOffice.org logók";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_RELEGION
+{
+Text [ en-US ] = "Relegion";
+Text [ de ] = "Religion";
+Text [ hu ] = "Vallás";
+Text [ x-comment ] = " ";
+};
+
+// ---------------------------------------------------------------------------
+
+String RID_GALLERYSTR_THEME_BUILDINGS
+{
+Text [ en-US ] = "Buildings";
+Text [ de ] = "Bau";
+Text [ hu ] = "Épületek";
+Text [ x-comment ] = " ";
+};
 
+// ---------------------------------------------------------------------------+
 
