--- scp2/source/ooo/module_hidden_ooo.scp.orig	2006-05-17 09:51:07.000000000 +0200
+++ scp2/source/ooo/module_hidden_ooo.scp	2006-07-05 07:22:38.703125000 +0200
@@ -901,6 +901,16 @@
 	gid_File_Res_Eme);
 End
 
+
+Module gid_Module_Root_Files_8_Lang_Res
+	ParentID = gid_Module_Root_Files_8;
+        Description = "gid_Module_Root_Files_8";
+	Default = YES;
+	Styles = (HIDDEN_ROOT);
+	Files = (gid_File_Res_Gal);
+End
+
+
 Module gid_Module_Root_Files_9
 	ParentID = gid_Module_Root;
 	Name = "gid_Module_Root_Files_9";
