--- svtools/source/svhtml/htmlout.cxx.orig	2005-10-20 21:35:08.515625000 +0200
+++ svtools/source/svhtml/htmlout.cxx	2005-10-21 19:07:09.062500000 +0200
@@ -312,6 +312,47 @@
 	case 9830:		pStr = sHTML_S_diams;	break;
 	}
 
+
+// No entities please in Latin-2 html
+	if( pStr && ( eDestEnc == RTL_TEXTENCODING_ISO_8859_2 || eDestEnc == RTL_TEXTENCODING_MS_1250 ))
+	{
+		if( !strcmp( pStr,"iacute" ) ||
+		    !strcmp( pStr,"Iacute" ) ||
+		    !strcmp( pStr,"yacute" ) ||
+		    !strcmp( pStr,"Yacute" ) ||
+		    !strcmp( pStr,"uuml" )   ||
+		    !strcmp( pStr,"Uuml" )   ||
+		    !strcmp( pStr,"ouml" )   ||
+		    !strcmp( pStr,"Ouml" )   ||
+		    !strcmp( pStr,"euml" )   ||
+		    !strcmp( pStr,"Euml" )   ||
+		    !strcmp( pStr,"auml" )   ||
+		    !strcmp( pStr,"Auml" )   ||
+		    !strcmp( pStr,"acirc" )  ||
+		    !strcmp( pStr,"Acirc" )  ||
+		    !strcmp( pStr,"ocirc" )  ||
+		    !strcmp( pStr,"Ocirc" )  ||
+		    !strcmp( pStr,"icirc" )  ||
+		    !strcmp( pStr,"Icirc" )  ||
+		    !strcmp( pStr,"ccedil" ) ||
+		    !strcmp( pStr,"Ccedil" ) ||
+		    !strcmp( pStr,"aacute" ) ||
+		    !strcmp( pStr,"Aacute" ) ||
+		    !strcmp( pStr,"eacute" ) ||
+		    !strcmp( pStr,"Eacute" ) ||
+		    !strcmp( pStr,"oacute" ) ||
+		    !strcmp( pStr,"Oacute" ) ||
+		    !strcmp( pStr,"scaron" ) ||
+		    !strcmp( pStr,"Scaron" ) ||
+		    !strcmp( pStr,"uacute" ) ||
+		    !strcmp( pStr,"Uacute" ) ||
+		    !strcmp( pStr,"divide" ) ||
+		    !strcmp( pStr,"curren" ) ||
+		    !strcmp( pStr,"times" )  ||
+		    !strcmp( pStr,"szlig" )) pStr = 0;
+	}
+
+
 	// Greek chars: if we do not produce a Greek encoding,
 	// transform them into entities
 	if( !pStr &&
