--- sc/source/ui/view/tabvwsh3.cxx.orig	2005-12-16 09:09:08.000000000 +0100
+++ sc/source/ui/view/tabvwsh3.cxx	2006-02-10 13:39:44.781250000 +0100
@@ -920,9 +920,9 @@
 
 		case SID_STATUS_DOCPOS:
 			{
-				//!	Navigator an-/ausschalten (wie im Writer) ???
-				//!GetViewData()->GetDispatcher().Execute( SID_NAVIGATOR,
-				//!						  SFX_CALLMODE_SYNCHRON|SFX_CALLMODE_RECORD );
+				//	Navigator an-/ausschalten (wie im Writer)
+				GetViewData()->GetDispatcher().Execute( SID_NAVIGATOR,
+										  SFX_CALLMODE_SYNCHRON|SFX_CALLMODE_RECORD );
 			}
 			break;
 
