--- sw/source/ui/frmdlg/cption.cxx.orig	2005-09-11 17:24:55.000000000 +0200
+++ sw/source/ui/frmdlg/cption.cxx	2005-10-21 19:49:24.187500000 +0200
@@ -427,11 +427,6 @@
                                         aFormatBox.GetSelectEntryPos() );
         if( SVX_NUM_NUMBER_NONE != nNumFmt )
         {
-            // Kategorie
-            aStr += sFldTypeName;
-            if ( aStr.Len() > 0 )
-                aStr += ' ';
-
             SwWrtShell &rSh = rView.GetWrtShell();
             String sFldTypeName( aCategoryBox.GetText() );
             SwSetExpFieldType* pFldType = (SwSetExpFieldType*)rSh.GetFldType(
@@ -460,6 +455,10 @@
             //case ARABIC:
             default:                    aStr += '1'; break;
             }
+            // Kategorie
+            aStr += '.';
+            aStr += ' ';
+            aStr += sFldTypeName;
         }
 
         aStr += aSepEdit.GetText();
--- sw/source/core/doc/doclay.cxx.orig	2005-09-11 17:23:48.000000000 +0200
+++ sw/source/core/doc/doclay.cxx	2005-10-21 20:07:02.078125000 +0200
@@ -1566,8 +1566,9 @@
         String aTxt;
         if(pType)
         {
-            aTxt += pType->GetName();
+            aTxt += '.';
             aTxt += ' ';
+            aTxt += pType->GetName();
         }
 		xub_StrLen nIdx = aTxt.Len();
         aTxt += rSeparator;
@@ -1584,7 +1584,7 @@
         if(pType)
         {
             SwSetExpField aFld( (SwSetExpFieldType*)pType, aEmptyStr, SVX_NUM_ARABIC);
-            pNew->Insert( SwFmtFld( aFld ), nIdx, nIdx );
+            pNew->Insert( SwFmtFld( aFld ), 0, 0 );
             if(rCharacterStyle.Len())
             {
                 SwCharFmt* pCharFmt = FindCharFmtByName( rCharacterStyle );
--- sw/source/ui/config/optload.cxx.orig	2005-11-12 18:59:30.000000000 +0100
+++ sw/source/ui/config/optload.cxx	2005-11-12 21:08:42.000000000 +0100
@@ -905,9 +905,6 @@
                                         aFormatBox.GetSelectEntryPos() );
         if( SVX_NUM_NUMBER_NONE != nNumFmt )
         {
-            // Kategorie
-            aStr += aCategoryBox.GetText();
-            aStr += ' ';
 
             SwWrtShell *pSh = ::GetActiveWrtShell();
             String sFldTypeName( aCategoryBox.GetText() );
@@ -940,6 +937,10 @@
                 //case ARABIC:
                 default:                    aStr += '1'; break;
             }
+            // Kategorie
+            aStr += '.';
+            aStr += ' ';
+            aStr += aCategoryBox.GetText();
         }
         aStr += aTextEdit.GetText();
     }
