--- svx/source/dialog/fntctrl.cxx.orig	2005-09-11 17:21:06.000000000 +0200
+++ svx/source/dialog/fntctrl.cxx	2005-10-15 19:33:41.915643100 +0200
@@ -34,6 +34,7 @@
  ************************************************************************/
 
 // include ---------------------------------------------------------------
+#include "chardlg.hrc"
 
 #ifndef _SFXVIEWSH_HXX
 #include <sfx2/viewsh.hxx>		// SfxViewShell
@@ -83,7 +84,7 @@
 
 #include "fntctrl.hxx"
 #include "dialogs.hrc"
-#define TEXT_WIDTH	20
+#define TEXT_WIDTH	50
 
 using namespace ::com::sun::star::uno;
 using namespace ::com::sun::star::lang;
@@ -611,6 +612,8 @@
 {
 	Printer* pPrinter = pImpl->pPrinter;
 	SvxFont& rFont = pImpl->aFont;
+	static sal_Char *str_fontpreview="külvízen úszó szárazjégtörő kisjármű";
+	UniString str_fontpreview_utf=UniString(ByteString(str_fontpreview),RTL_TEXTENCODING_UTF8);
 
     if ( pImpl->bUseResText )
 		pImpl->aText = GetText();
@@ -627,7 +630,10 @@
 		}
 
 		if ( !pImpl->bSelection )
-			pImpl->aText = rFont.GetName();
+//			pImpl->aText = rFont.GetName();
+			pImpl->aText = str_fontpreview_utf;
+//			pImpl->aText.Append(rFont.GetName());
+
 
 		if ( !pImpl->aText.Len() )
 			pImpl->aText = GetText();
