--- svx/source/editeng/svxacorr.cxx.orig	2005-09-11 17:21:38.000000000 +0200
+++ svx/source/editeng/svxacorr.cxx	2005-10-15 19:37:29.462518100 +0200
@@ -718,7 +718,26 @@
 		}
 	}
 
-	// ersetze [A-z0-9]--[A-z0-9] durch "emDash"
+	// ersetze [A-z0-9]---[A-z0-9] durch "emDash"
+	if( cEmDash && 4 <= nEndPos - nSttPos )
+	{
+		String sTmp( rTxt.Copy( nSttPos, nEndPos - nSttPos ) );
+		xub_StrLen nFndPos = sTmp.SearchAscii( "---" );
+		if( STRING_NOTFOUND != nFndPos && nFndPos &&
+			nFndPos + 3 < sTmp.Len() &&
+			( rCC.isLetterNumeric( sTmp, nFndPos - 1 ) ||
+			  lcl_IsInAsciiArr( sImplEndSkipChars, rTxt.GetChar( nFndPos - 1 ) )) &&
+			( rCC.isLetterNumeric( sTmp, nFndPos + 3 ) ||
+			lcl_IsInAsciiArr( sImplSttSkipChars, rTxt.GetChar( nFndPos + 3 ) )))
+		{
+			nSttPos += nFndPos;
+			rDoc.Delete( nSttPos, nSttPos + 3 );
+			rDoc.Insert( nSttPos, cEmDash );
+			bRet = TRUE;
+		}
+	}
+
+	// ersetze [A-z0-9]--[A-z0-9] durch "enDash"
 	if( cEmDash && 4 <= nEndPos - nSttPos )
 	{
 		String sTmp( rTxt.Copy( nSttPos, nEndPos - nSttPos ) );
@@ -732,7 +751,7 @@
 		{
 			nSttPos += nFndPos;
 			rDoc.Delete( nSttPos, nSttPos + 2 );
-			rDoc.Insert( nSttPos, cEmDash );
+			rDoc.Insert( nSttPos, cEnDash );
 			bRet = TRUE;
 		}
 	}
