--- scp2/source/ooo/file_jre_ooo.scp.original	2006-08-11 23:21:34.000000000 +0200
+++ scp2/source/ooo/file_jre_ooo.scp	2006-10-12 19:54:42.167371800 +0200
@@ -39,7 +39,7 @@
     BIN_FILE_BODY;
 	Styles = (DONT_PACK,JAVAFILE);
 	Dir = gid_Dir_Program;
-	Name = "jre-1_5_0_07-windows-i586-p.exe";
-	Javaversion = "1.5.0_07";
+	Name = "jre-1_5_0_09-windows-i586-p.exe";
+	Javaversion = "1.5.0_09";
 	Subdir = "java";
 End
