--- ./svx/source/intro/makefile.mk.orig	2005-12-16 13:26:18.000000000 +0100
+++ ./svx/source/intro/makefile.mk	2005-01-15 08:15:06.234375000 +0100
@@ -46,7 +46,7 @@
 RSCLOCINC!:=$(RSCLOCINC);$(PRJ)$/RES
 
 # setting the compiled by $user$ string in the About box
-OOO_VENDOR*=$(USER)$(USERNAME)
+OOO_VENDOR*=FSF.hu Alapítvány
 RSCDEFS+=-DOOO_VENDOR="$(OOO_VENDOR)"
 
 # --- Allgemein -----------------------------------------------------------
