--- officecfg/registry/data/org/openoffice/Office/UI/WriterGlobalWindowState.xcu.orig	2006-08-11 21:53:31.000000000 +0200
+++ officecfg/registry/data/org/openoffice/Office/UI/WriterGlobalWindowState.xcu	2006-10-19 14:23:01.203125000 +0200
@@ -688,6 +688,31 @@
 					<value>true</value>
 				</prop>
 			</node>
+						<node oor:name="private:resource/toolbar/dataminer" oor:op="replace">
+				<prop oor:name="DockPos" oor:type="xs:string">
+					<value>1,1</value>
+				</prop>
+				<prop oor:name="Docked" oor:type="xs:boolean">
+					<value>true</value>
+				</prop>
+				<prop oor:name="DockingArea" oor:type="xs:int">
+					<value>0</value>
+				</prop>
+				<prop oor:name="UIName" oor:type="xs:string">
+					<value xml:lang="en-US">Dataminer</value>
+					<value xml:lang="de">Angabe Knappe</value>
+					<value xml:lang="hu">Adatbányász</value>
+				</prop>
+				<prop oor:name="Visible" oor:type="xs:boolean">
+					<value>true</value>
+				</prop>
+				<prop oor:name="HideFromToolbarMenu" oor:type="xs:boolean">
+					<value>false</value>
+				</prop>
+				<prop oor:name="ContextSensitive" oor:type="xs:boolean">
+				    <value>false</value>
+				</prop>
+			</node>
 		</node>
 	</node>
 </oor:component-data>

--- officecfg/registry/data/org/openoffice/Office/UI/WriterWebWindowState.xcu.orig	2006-08-11 21:53:31.000000000 +0200
+++ officecfg/registry/data/org/openoffice/Office/UI/WriterWebWindowState.xcu	2006-10-19 14:23:01.203125000 +0200
@@ -615,6 +615,31 @@
 					<value>true</value>
 				</prop>
 			</node>
+						<node oor:name="private:resource/toolbar/dataminer" oor:op="replace">
+				<prop oor:name="DockPos" oor:type="xs:string">
+					<value>1,1</value>
+				</prop>
+				<prop oor:name="Docked" oor:type="xs:boolean">
+					<value>true</value>
+				</prop>
+				<prop oor:name="DockingArea" oor:type="xs:int">
+					<value>0</value>
+				</prop>
+				<prop oor:name="UIName" oor:type="xs:string">
+					<value xml:lang="en-US">Dataminer</value>
+					<value xml:lang="de">Angabe Knappe</value>
+					<value xml:lang="hu">Adatbányász</value>
+				</prop>
+				<prop oor:name="Visible" oor:type="xs:boolean">
+					<value>true</value>
+				</prop>
+				<prop oor:name="HideFromToolbarMenu" oor:type="xs:boolean">
+					<value>false</value>
+				</prop>
+				<prop oor:name="ContextSensitive" oor:type="xs:boolean">
+				    <value>false</value>
+				</prop>
+			</node>
 		</node>
 	</node>

--- officecfg/registry/data/org/openoffice/Office/UI/WriterWindowState.xcu.orig         2006-08-11 21:53:31.000000000 +0200
+++ officecfg/registry/data/org/openoffice/Office/UI/WriterWindowState.xcu              2006-10-19 14:23:01.203125000 +0200
@@ -694,6 +694,31 @@
 					<value>true</value>
 				</prop>
 			</node>
+						<node oor:name="private:resource/toolbar/dataminer" oor:op="replace">
+				<prop oor:name="DockPos" oor:type="xs:string">
+					<value>1,1</value>
+				</prop>
+				<prop oor:name="Docked" oor:type="xs:boolean">
+					<value>true</value>
+				</prop>
+				<prop oor:name="DockingArea" oor:type="xs:int">
+					<value>0</value>
+				</prop>
+				<prop oor:name="UIName" oor:type="xs:string">
+					<value xml:lang="en-US">Dataminer</value>
+					<value xml:lang="de">Angabe Knappe</value>
+					<value xml:lang="hu">Adatbányász</value>
+				</prop>
+				<prop oor:name="Visible" oor:type="xs:boolean">
+					<value>true</value>
+				</prop>
+				<prop oor:name="HideFromToolbarMenu" oor:type="xs:boolean">
+					<value>false</value>
+				</prop>
+				<prop oor:name="ContextSensitive" oor:type="xs:boolean">
+				    <value>false</value>
+				</prop>
+			</node>
 		</node>
 	</node>

