--- solenv/bin/modules/installer/languagepack.pm.orig	2006-05-17 09:44:16.000000000 +0200
+++ solenv/bin/modules/installer/languagepack.pm	2006-06-07 22:58:57.248250000 +0200
@@ -4,9 +4,9 @@
 #
 #   $RCSfile: languagepack.pm,v $
 #
-#   $Revision: 1.10 $
+#   $Revision: 1.11 $
 #
-#   last change: $Author: rt $ $Date: 2005/09/08 09:04:19 $
+#   last change: $Author: KAMI $ $Date: 2006/06/07 22:04:19 $
 #
 #   The Contents of this file are made available subject to
 #   the terms of GNU Lesser General Public License Version 2.1.
@@ -93,6 +93,8 @@
 						if ( $oneitem->{'Dir'} eq "gid_Dir_Fonts_Truetype" ) { $oneitem->{'modules'} = "gid_Module_Langpack_Fonts"; }
 						if ( $oneitem->{'Dir'} eq "gid_Dir_Resource" ) { $oneitem->{'modules'} = "gid_Module_Langpack_Resource"; }
 						if ( $oneitem->{'Dir'} eq "gid_Dir_Help_Isolanguage" ) { $oneitem->{'modules'} = "gid_Module_Langpack_Help"; }
+						if ( $oneitem->{'Dir'} eq "gid_Dir_Samples_Language" ) { $oneitem->{'modules'} = "gid_Module_Langpack_Accessories_Samples"; }
+						if ( $oneitem->{'Dir'} eq "gid_Dir_Template_Language" ) { $oneitem->{'modules'} = "gid_Module_Langpack_Accessories_Templates"; }
 					}
 				}
 
@@ -194,10 +196,11 @@
 
 	for ( my $i = 0; $i <= $#{$alldirs}; $i++ )
 	{
-		if ( ${$alldirs}[$i] =~ /-fonts/ ) { next; } 
-		if ( ${$alldirs}[$i] =~ /-help/ ) { next; } 
-		if ( ${$alldirs}[$i] =~ /-res/ ) { next; } 
-	
+		if ( ${$alldirs}[$i] =~ /-fonts/ ) { next; }
+		if ( ${$alldirs}[$i] =~ /-help/ ) { next; }
+		if ( ${$alldirs}[$i] =~ /-res/ ) { next; }
+        	if ( ${$alldirs}[$i] =~ /-premium-samples/ ) { next; }
+		if ( ${$alldirs}[$i] =~ /-premium-templates/ ) { next; }
 		$packagename = ${$alldirs}[$i];
 		last;
 	}
