--- instsetoo_native/inc_openoffice/windows/msi_templates/InstallE.idt.orig	2006-05-17 09:51:41.000000000 +0200
+++ instsetoo_native/inc_openoffice/windows/msi_templates/InstallE.idt	2006-05-29 21:07:09.109375000 +0200
@@ -51,7 +51,6 @@
 RemoveRegistryValues		1300
 RemoveShortcuts		1600
 RMCCPSearch	Not CCP_SUCCESS And CCP_TEST	250
-SameProductFound	SAMEPRODUCTS	120
 ScheduleReboot	ISSCHEDULEREBOOT	3125
 SelfRegModules		2850
 SelfUnregModules		1100
