--- wizards/prj/build.lst.orig	2006-05-17 09:41:42.000000000 +0200
+++ wizards/prj/build.lst	2006-07-02 13:53:39.218750000 +0200
@@ -13,6 +13,8 @@
 wz      wizards\source\tutorials                                nmake           -       all     wz_tutorials NULL
 wz      wizards\source\standard                                 nmake           -       all     wz_standard NULL
 wz      wizards\source\launcher                                 nmake           -       all     wz_launcher NULL
+wz      wizards\source\premium                                  nmake           -       all     wz_premium NULL
+wz      wizards\source\ooowikipedia                             nmake           -       all     wz_ooowikipedia NULL
 wz      wizards\com\sun\star\wizards                            nmake           -       all     wz_commonwiz NULL
 wz      wizards\com\sun\star\wizards\query                      nmake           -       all     wz_querywiz wz_commonwiz NULL
 wz      wizards\com\sun\star\wizards\report                     nmake           -       all     wz_reportwiz wz_commonwiz NULL
