--- filter/source/config/fragments/fcfg_drawgraphics.mk.orig	2006-05-17 09:48:19.000000000 +0200
+++ filter/source/config/fragments/fcfg_drawgraphics.mk	2006-07-04 18:24:34.375000000 +0200
@@ -34,7 +34,7 @@
     xpm_XPM
 
 # -----------------------------------------------
-# count = 45
+# count = 46
 F4_DRAWGRAPHICS = \
     BMP___MS_Windows \
     DXF___AutoCAD_Interchange \
@@ -77,6 +77,7 @@
     draw_ppm_Export \
     draw_ras_Export \
     draw_svg_Export \
+    draw_svg_Import \
     draw_svm_Export \
     draw_tif_Export \
     draw_wmf_Export \
