--- filter/source/config/fragments/fcfg_impressgraphics.mk.orig	2006-05-17 09:48:19.000000000 +0200
+++ filter/source/config/fragments/fcfg_impressgraphics.mk	2006-07-04 18:24:18.578125000 +0200
@@ -25,7 +25,7 @@
     xpm_XPM
 
 # -----------------------------------------------
-# count = 20
+# count = 21
 F4_IMPRESSGRAPHICS = \
     CGM___Computer_Graphics_Metafile \
     impress_bmp_Export \
@@ -43,6 +43,7 @@
     impress_ppm_Export \
     impress_ras_Export \
     impress_svg_Export \
+    draw_svg_Import \
     impress_svm_Export \
     impress_tif_Export \
     impress_wmf_Export \
