--- filter/source/config/fragments/types/svg_Scalable_Vector_Graphics.xcu.orig	2006-05-17 09:48:20.000000000 +0200
+++ filter/source/config/fragments/types/svg_Scalable_Vector_Graphics.xcu	2006-07-04 18:09:14.359375000 +0200
@@ -1,12 +1,11 @@
 	<node oor:name="svg_Scalable_Vector_Graphics" oor:op="replace" >
-		<prop oor:name="DetectService"/>
+		<prop oor:name="DetectService"><value>haui.office.uno.mediadetect</value></prop>
 		<prop oor:name="URLPattern"/>
 		<prop oor:name="Extensions"><value>svg</value></prop>
 		<prop oor:name="MediaType"><value>image/svg+xml</value></prop>
-		<prop oor:name="Preferred"><value>false</value></prop>
-		<prop oor:name="PreferredFilter"/>
-		<prop oor:name="UIName">
-			<value>SVG - Scalable Vector Graphics</value>
-		</prop>
+		<prop oor:name="Preferred"><value>true</value></prop>
+		<prop oor:name="PreferredFilter"><value>draw_svg_Import</value></prop>
+		<prop oor:name="UIName"><value>SVG - Scalable Vector Graphics</value></prop>
+		<prop oor:name="UIOrder"><value>0</value></prop>		
 		<prop oor:name="ClipboardFormat"/>
 	</node>
